packages/server/lib/makeDataContext.ts

```diff
@@ -164,6 +164,9 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
       async runSpec (spec: Cypress.Spec): Promise<void> {
         openProject.changeUrlToSpec(spec)
       },
+      async routeToDebug (): Promise<void> {
```

Comment:
Doesn't seem like this needs to be async. I'm assuming you patterned it off the `runSpec` function above which also doesn't seem to need it

Reply:
Updated.